R/net.inputs.R

```diff
@@ -658,6 +658,18 @@ crosscheck.net <- function(x, param, init, control) {
         }
       }
 
+      # Checks that .m2 syntax update to .g2
```

Comment:
Why is this done in crosscheck and not in param.net and init.net?

Reply:
Checks moved to param and init respectively. Updated unit tests.